safekeeper/src/debug_dump.rs

```diff
@@ -140,8 +202,12 @@ pub async fn build(args: Args) -> Result<Response> {
         GlobalTimelines::get_all()
     };
 
-    // TODO: return Stream instead of Vec
     let mut timelines = Vec::new();
+    let runtime = Arc::new(
+        tokio::runtime::Builder::new_current_thread()
```

Comment:
Interesting, I wonder how does it work. I didn't thought that Rust allows sending `new_current_thread()` to other threads, but apparently it does...